backend/src/dao/DevPortfolioDao.ts

```diff
@@ -0,0 +1,64 @@
+import { v4 as uuidv4 } from 'uuid';
+import { devPortfolioCollection } from '../firebase';
+import { DBDevPortfolio } from '../types/DataTypes';
+import { getMemberFromDocumentReference } from '../utils/memberUtil';
+
+export default class DevPortfolioDao {
+  static async makeDevPortfolioSubmission(
+    uuid: string,
+    submission: DevPortfolioSubmission
+  ): Promise<DevPortfolioSubmission> {
+    const doc = await devPortfolioCollection.doc(uuid).get();
+
+    const data = doc.data() as DBDevPortfolio;
+
+    const subs = data.submissions;
+    subs.push(submission);
+    await devPortfolioCollection.doc(uuid).update({ submissions: subs });
```

Comment:
submission should be of type `DevPortfolioSubmission` and needs to be converted to `DBDevPortfolioSubmission`